### PR TITLE
updates http request and response  of graphql query input testcase.

### PR DESCRIPTION
**Pull request by re-Tick**
## Related Issue
  - Info about Issue or bug

Closes: #138 
#### Describe the changes you've made
adds update function only updates http request and response of graphql query input testcase
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

File: graph/utils.go [hunk 1]
```diff
@@ -87,6 +87,7 @@ func ConvertHttpReq(r models.HttpReq) *model.HTTPReq {
 		Header:     header,
 		Method:     ConvertMethod(r.Method),
 		Body:       r.Body,
+		URL:        &r.URL,
```
Comment (slayerjain) [inline graph/utils.go:90]: this seems to be redundant from your previous PR.

File: graph/utils.go [hunk 2]
```diff
@@ -198,6 +199,9 @@ func ConvertTestCaseInput(input *model.TestCaseInput) models.TestCase {
 		if input.HTTPReq.Body != nil {
 			req.Body = *input.HTTPReq.Body
 		}
+		if input.HTTPReq.URL != nil {
```
Comment (slayerjain) [inline graph/utils.go:202]: this too

**Review by slayerjain — DISMISSED** [review]
please review inline comments. Also please add few testcases to avoid decreasing the coverage.

**Comment by nehagup** [comment]
@re-Tick We can add normalise test case now, since it's fixed :D

File: graph/utils.go (hunk 1 above)
Comment (re-Tick) [reply graph/utils.go]: Done

**Comment by nehagup** [comment]
@re-Tick we need to rebase this branch from main. With your latest commit it'll remove the URL field again :)

**Comment by re-Tick** [comment]
> @re-Tick we need to rebase this branch from main. With your latest commit it'll remove the URL field again :)

I think that will change will not be discarded from main.

**Review by nehagup — APPROVED** [review]
LGTM

**Pull request merged**